### DISCUSSION TITLE
Which do you want to have in next big release?

### DISCUSSION THREAD

**Discussion by tim-hub** [tim-hub/obsidian-bible-reference#42] [Polls]
Dear OBR Users, and Bible reader/researcher/lovers

It has been about 9 months since the 1st release of Obsidian Bible Reference. (OBR).

Without your feedback, it wouldn't be at current stage.

**Now OBR gets**
- **1988 downloads**
![image](https://user-images.githubusercontent.com/2884320/178081560-732a8ace-2f96-426c-a778-32a1ad8a7441.png)

- [**10 releases** since November 2021](https://github.com/tim-hub/obsidian-bible-reference/releases/tag/0.1.2)
- [**7**/18 issues resolved](https://github.com/tim-hub/obsidian-bible-reference/issues)

**I know OBR still has** 
- bugs
- features not been achieved

Thanks you again for your patience and help. As a full time software engineer and part time Open Source contributor, it is not easy for me to manage time to give OBR more time, so I really want to know what would be the top feature or bug (to fix) you prefer?

See more https://github.com/tim-hub/obsidian-bible-reference/issues

**Comment by jmantn** [comment]
Offline mode?  For taking notes during service i have my iPad on silent focus and not connected to the internet.
